src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerNegativeTests.cs

```diff
@@ -51,16 +54,45 @@ public async Task GetInvitation_AsViewer_ShouldReturnNotFound_WhenUnknownId()
                 HttpStatusCode.NotFound);
         #endregion
 
-        #region Create
+        #region GetInvitationsByCommPkgNo
+        [TestMethod]
+        public async Task GetInvitationsByCommPkgNo_AsAnonymous_ShouldReturnUnauthorized()
+            => await InvitationsControllerTestsHelper.GetInvitationsByCommPkgNoAsync(
+                UserType.Anonymous,
+                TestFactory.UnknownPlant,
+                "CommPkgNo1",
+                TestFactory.ProjectWithAccess,
+                HttpStatusCode.Unauthorized);
 
+        [TestMethod]
+        public async Task GetInvitationsByCommPkgNo_AsHacker_ShouldReturnBadRequest_WhenUnknownPlant()
+            => await InvitationsControllerTestsHelper.GetInvitationsByCommPkgNoAsync(
+                UserType.Hacker,
+                TestFactory.UnknownPlant,
+                "CommPkgNo1",
+                TestFactory.ProjectWithAccess,
+                HttpStatusCode.BadRequest,
+                "is not a valid plant");
+
+        [TestMethod]
+        public async Task GetInvitationsByCommPkgNo_AsHacker_ShouldReturnForbidden_WhenPermissionMissing()
+            => await InvitationsControllerTestsHelper.GetInvitationsByCommPkgNoAsync(
+                UserType.Hacker,
+                TestFactory.PlantWithAccess,
+                "CommPkgNo1",
+                TestFactory.ProjectWithAccess,
+                HttpStatusCode.Forbidden);
+        #endregion
+
+        #region Create
```

Comment:
I think more negative tests regarding permissions should be added:
* Create should fail if signer
* Create should fail if one of accepter or completer

Reply:
Fail if signer exists already, will add for the other two